src/wsiToDcm.cpp

```diff
@@ -172,6 +220,7 @@ std::unique_ptr<SlideLevelDim> WsiToDcm::getSlideLevelDim(const int32_t level,
       downsampleOfLevel = static_cast<double>(downsample) / multiplicator;
       // check that downsampling is going from higher to lower magnification
       if (downsampleOfLevel >= 1.0) {
+        levelToGet = priorLevel->level;
```

Comment:
levelToGet and the other parameters are set here when the slide is being progressive downsampled.  generateUsingOpenSlide = false so the next if loop won't be entered.  Some of these are not actually used if progressive downsampling is being used. I set them here largely for completeness to a avoid undefined weirdness.

Yes, your comment, about level zero made me realize that the downsampling list enabled downsamples to be ordered in any order.  This would have broken the code.  The most recent CL should address this edge case and add unit tests to test it. Its not something that my usage of the code needs. But, should make it complete for others.